### PR TITLE
Input Next docs

### PR DESCRIPTION
**Pull request by libertymayc**
Docs for Input Next
cc @bhoppers2008

### CODE REVIEW

File: packages/lab/stories/input-next/input-next.doc.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Input provides an editable field in which users can enter arbitrary text and numeric values.
+
+# Examples
+
+## Default
+
+Input comprises value text and a focus ring. The component is initially empty and then, once it has focus, it accepts text on a single line only.
+
+<Canvas>
+  <Story id="core-input-next--default" />
```
Comment (feliciackh) [inline packages/lab/stories/input-next/input-next.doc.stories.mdx:29]:
```suggestion
  <Story id="lab-input-next--default" />
```

& others too

File: packages/lab/stories/input-next/input-next.doc.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  <Story id="core-input-next--validation" />
+</Canvas>
+
+- Use the error state to alert the user of a critical issue that’s related to the input. This issue, which may jeopardize completion of the task, usually requires action from the user to resolve the error.
+- Display the warning state when you need to alert the user of a potential issue—that won’t prevent the user from continuing with the task, but may cause errors if it’s not addressed.
+
+# API
+
+```
+import { InputNext } from "@salt-ds/lab";
```
Comment (ZarkoPernar) [inline packages/lab/stories/input-next/input-next.doc.stories.mdx:92]: This should be at the beginning, no?

**Review by ZarkoPernar — APPROVED** [review]

File: packages/lab/stories/input-next/input-next.doc.stories.mdx (hunk 1 above)
Comment (libertymayc) [reply packages/lab/stories/input-next/input-next.doc.stories.mdx]: thank you!

File: packages/lab/stories/input-next/input-next.doc.stories.mdx (hunk 2 above)
Comment (libertymayc) [reply packages/lab/stories/input-next/input-next.doc.stories.mdx:92]: hmm, it's in this position in all other docs + template

Comment (ZarkoPernar) [reply packages/lab/stories/input-next/input-next.doc.stories.mdx:92]: Eesh, ok, then leave it as is 👍

Comment (libertymayc) [reply packages/lab/stories/input-next/input-next.doc.stories.mdx:92]: worth raising it with the content team :)

**Review by alycrys — DISMISSED** [review]

**Review by feliciackh — APPROVED** [review]

**Review by feliciackh — APPROVED** [review]

File: packages/lab/stories/input-next/input-next.doc.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+Input comprises value text and a focus ring. The component is initially empty and then, once it has focus, it accepts text on a single line.
+
+<Canvas>
+  <Story id="lab-input-next--default" />
+</Canvas>
+
+## Controlled
+
+You can control Input with the `value` and `onChange` props. Learn more about <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">controlled vs. uncontrolled components in the official React docs</Link>.
```
Comment (joshwooding) [inline packages/lab/stories/input-next/input-next.doc.stories.mdx:34]: Minor: I would expect the Link to wrap a different part of this sentence but maybe worth checking?

e.g.

```suggestion
You can control Input with the `value` and `onChange` props. Learn more about <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">controlled vs. uncontrolled components</Link> in the official React docs.
```

```suggestion
You can control Input with the `value` and `onChange` props. Learn more about controlled vs. uncontrolled components in the <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">official React docs</Link>.
```

File: packages/lab/stories/input-next/input-next.doc.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+For accessibility purposes, we recommend that you add a marker for empty read-only fields via the `emptyReadOnlyMarker` prop.
+
+<Canvas>
+  <Story id="lab-input—next--empty-readonly-marker" />
+</Canvas>
+
+## Text Alignment
+
+Text alignment is left by default. This can be changed via the `textAlign` prop to “center” or “right”.
```
Comment (joshwooding) [inline packages/lab/stories/input-next/input-next.doc.stories.mdx:72]: This is a minor thing, but are we using ” and ’ everywhere now?

File: packages/lab/stories/input-next/input-next.doc.stories.mdx (hunk 2 above)
Comment (joshwooding) [reply packages/lab/stories/input-next/input-next.doc.stories.mdx:92]: I think it's fine to keep it here so it aligns to the template

File: packages/lab/stories/input-next/input-next.doc.stories.mdx (hunk 4 above)
Comment (sion-docs) [reply packages/lab/stories/input-next/input-next.doc.stories.mdx:72]: That is the current preferred style.

**Review by alycrys — APPROVED** [review]

**Pull request merged**